packages/web-components/fast-foundation/src/combobox/combobox.spec.md

```diff
@@ -0,0 +1,149 @@
+  - `inline`: Automatically fills the value of the first matching option while typing, and highlights the auto-inserted text. While a value is present, navigating through the list will automatically jump to partially-matching options.
+  - `list`: Auto-filters the available options to only display those which start with the current value.
+  - `both`: Automatically filters the list and fills the remaining text for the currently selected option.
+- **Opened and closed states**: The `combobox` provides an expandable listbox can be displayed by clicking on the element, or using keyboard navigation.
+- **Positioning**: When the `combobox` does not have enough screen real estate to open below the control, it will open above.
+- **Single and multiple selection mode**: Users can choose one or multiple options when the `multiple` attribute is present. *(Note: While our implementation currently only supports single selection mode, multiple selection mode is being tracked in [issue #4190](https://github.com/microsoft/fast/issues/4190).)*
+- **Keyboard navigation and type-ahead**: When the `combobox` is focused, keyboard navigation with the arrow keys will cycle through the available options. Type-ahead is also supported. See [Combobox Keyboard Interaction](https://w3c.github.io/aria-practices/#combobox-keyboard-interaction) for more details.
+
+### Prior Art/Examples
+
+- [Ant Design](https://ant.design/components/auto-complete/)
+- [Carbon Design](https://www.carbondesignsystem.com/components/select/code/)
+- [Atlassian UI](https://atlaskit.atlassian.com/packages/core/select)
+- [Lightning Design System](https://www.lightningdesignsystem.com/components/combobox/)
+- [W3 Example](https://w3c.github.io/aria-practices/examples/combobox/combobox-autocomplete-both.html)
+
+---
+
+## API
+
+Extends [`listbox`](../listbox/listbox.spec.md) and [form associated custom element](../form-associated/form-associated-custom-element.md).
+
+*Component Name*:
+
+- `fast-combobox`
+
+*Attributes*:
+
+- `autocomplete` - Handles autocomplete features for the control on pageload. Accepted values are `none`, `inline`, `list`, and `both`.
+- `disabled` - Disables the control.
+- `multiple` - Allows the user to select more than one option.
+- `name` - Name of the control.
+- `position` - The placement for the listbox when the combobox is open. Values may be either `above` or `below`.
+- `required` - Boolean value that sets the field as required.
+- `value` - The initial value of the combobox.
+
+*Properties*:
+
+- `options` - An array of all options.
+- `placeholder` - Sets the placeholder of the element, generally used to provide a hint to the user.
+- `role` - The role of the element.
+- `value` - Reflects the value of the first selected option. Setting the `value` property will update the `selected` state of the first option that matches the value, if any.
+
+*Events*:
+
+- `change` - Emits when the `value` is changed via user interaction.
```

Comment:
should it also emit an `input` event on input?

Reply:
An `input` event is emitted from the internal input, but I wasn't sure if the text box's native events should be considered as part of the spec here. I can add it if that is the case.

Reply:
I think it could be valuable to add it to the spec, if only for clarity.